ui/src/data/source.tsx

```diff
@@ -2141,13 +2256,30 @@ function fromAPICriteriaGroupSections(
     id: section.id,
     name: section.displayName,
     filter: {
-      kind:
-        section.operator === tanagra.CriteriaGroupSectionOperatorEnum.And
-          ? GroupSectionFilterKind.All
-          : GroupSectionFilterKind.Any,
+      kind: fromAPICriteriaGroupSectionOperator(section.operator),
       excluded: section.excluded,
     },
-    groups: section.criteriaGroups.map((group) => ({
+    groups: fromAPICriteriaGroups(underlaySource, section.criteriaGroups),
+    secondBlockGroups: fromAPICriteriaGroups(
+      underlaySource,
+      section.secondBlockCriteriaGroups
+    ),
+    operatorValue: section.operatorValue ?? 3,
```

Comment:
Is 3 the default value for number of days?

Reply:
Yeah